src/components/dev-hub/codeblock.js

```diff
@@ -41,17 +41,16 @@ const StyledCode = styled(Code)`
         }px)`};
     /* Line Numbers */
     > div {
-        background-color: ${colorMap.greyDarkTwo};
+        background-color: ${({ theme }) => theme.colorMap.greyDarkTwo};
         border-image: linear-gradient(
                 0deg,
-                ${colorMap.sherbet} 0%,
-                ${colorMap.salmon} 49.99%,
-                ${colorMap.magenta} 100%
+                ${({ theme }) => theme.colorMap.sherbet} 0%,
+                ${({ theme }) => theme.colorMap.salmon} 49.99%,
+                ${({ theme }) => theme.colorMap.magenta} 100%
             )
             1;
         border-width: 0 2px 0 0;
         border-right-style: solid;
-        color: ${colorMap.greyLightTwo};
```

Comment:
Let's add this back please!

Reply:
Oops! my bad. Must've removed it while resolving merge conflict.